skr-webhook/resources.yaml

```diff
@@ -92,7 +92,7 @@ spec:
               value: "v1"
             - name: KCP_ADDR
               value: "kcp-base-url-invalid"
-          image: "europe-docker.pkg.dev/kyma-project/prod/runtime-watcher-skr:0.1.1"
+          image: "europe-docker.pkg.dev/kyma-project/prod/runtime-watcher:1.1.5"
```

Comment:
https://github.com/kyma-project/lifecycle-manager/blob/main/config/watcher/kustomization.yaml#L17-L21

here is where we actually set this. Should we somehow make this clear by removing this line or setting it to a placeholder value?

Reply:
No idea how to do this quickly now since we want to release today. But maybe worth an investigation issue?